lib/internal/structured_clone.js

```diff
@@ -0,0 +1,23 @@
+'use strict';
+
+const {
+  MessageChannel,
+  receiveMessageOnPort
+} = require('internal/worker/io');
+
+let channel;
+function structuredClone(value, transfer) {
+  // TODO: Improve this with a more efficient solution that avoids
+  // instantiating a MessageChannel
+  if (!channel) {
+    channel = new MessageChannel();
+  }
```

Comment:
Is lazy-instantiating `MessageChannel` really needed?

Reply:
This was added based on @targos comment

Reply:
> [...] (lazily if needed)?

So it can also be
```js
// TODO: Improve this with a more efficient solution that avoids
// instantiating a MessageChannel
const channel = new MessageChannel();
function structuredClone(value, transfer) {
```
if there's no need for lazy-instantiating.

Reply:
I got a compiler error when I had it like this. Something about how the message channel can't be instantiated until during run time